cli/azd/pkg/ext/hooks_runner.go

```diff
@@ -159,6 +159,14 @@ func (h *HooksRunner) execHook(ctx context.Context, hookConfig *HookConfig, opti
 		defer h.console.StopPreviewer(ctx, false)
 	}
 
+	isRunWithNoProfile := false
```

Comment:
nit 

```suggestion
	var isRunWithNoProfile bool
```

Reply:
@vhvb1989 Got it. I've submitted new commits based on your comment. Please review it, thanks.